deviceModeInit.js

```diff
@@ -2,9 +2,7 @@ var rudderTracking = (function () {
   const pages = {
     "/products/": "Product Viewed",
     "/cart": "Cart Viewed",
-    "/collections/": "Product List Viewed",
     "/account/register": "Registration Viewed",
-    "/thank_you": "Checkout Step Completed",
```

Comment:
We are anyhow not getting any `/thank_you`  page for which this can trigger and in ecom spec we are already planning to map `Checkout Step Completed` with other Events